emulators/spim/Portfile

```diff
@@ -1,9 +1,9 @@
 PortSystem 1.0
 name            spim
-version         9.1.13
+version         9.1.20
 license         BSD
 categories      emulators devel lang x11
-maintainers     nomaintainer
+maintainers     {mails.ucas.ac.cn:chenguokai17 @chenguokai}
```

Comment:
Is there a good reason not to make this `openmaintainer` as well?

Reply:
I wanted to add this but didn’t know exactly how to add this option. The guide didn’t express it with an example

Reply:
Just add it to the end:

`maintainers     {mails.ucas.ac.cn:chenguokai17 @chenguokai} openmaintainer`